### ISSUE TITLE
Update to cordova@6.4.0

### ISSUE THREAD

**Issue by mwbrooks** [phonegap/phonegap-cli#632]
Cordova CLI 6.3.0 is now available:
http://cordova.apache.org/news/2016/07/13/tools-release.html

**Comment by surajpindoria** [comment]
Going to put this on hold for now. It requires changes to PhoneGap template apps which in turn requires changes in the desktop app. Current plan is to time this along with template support for desktop.

**Affected Templates**

- phonegap-template-hello-world
- phonegap-template-push
- phonegap-app-star-track

**Comment by Kjens93** [comment]
I suggest we modify this issue to support cordova@6.**4**.0 and we prioritize it because phonegap-plugin-push@1.9.0 (with support for iOS 10) is broken until phonegap begins requiring cordova@6.4.0.

**Comment by surajpindoria** [comment]
Hi @Kjens93,

I was just taking a look at this yesterday and was planning on using Cordova 6.4.0 as the issue is outdated now. We were just waiting on the latest release of the PhoneGap Desktop app before moving forward with this. That release is done so this issue is no longer blocked.
